task/0.1/tests/ecp-policy.yaml

```diff
@@ -21,10 +21,12 @@ metadata:
   name: ec-policy
 spec:
   description: My custom enterprise contract policy configuration
-  sources:
-    - policy:
-      - quay.io/hacbs-contract/ec-release-policy:latest
   configuration:
     exclude:
       - not_useful
```

Comment:
Let's remove it, we should not need it any more
```suggestion
```

Reply:
Thanks,removed.